### PR TITLE
awesome es6 refactoring

### PR DESCRIPTION
**Pull request by jpkempf**
got rid of jquery and bootstrap js, refactored code to native es6

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -40,6 +36,7 @@
         },
         {
             title:"JavaScript",
+            text:"The only programming language that matters.",
```
Comment (mrVanDalo) [inline docs/index.html]: hmm My intention here was not to rate languages. (Also it's wrong). please remove

File: docs/index.html [hunk 2]
```diff
@@ -132,43 +129,53 @@
...
-                        <div class="caption" style="text-align:center;font-weight:bold"> ${elem.label} </div>
-                      </a>
-                    </div>
-                    `
-                    )
-        }
+    /* --- */
+
+    const createItemRow = item => {
+        const title = `
```
Comment (mrVanDalo) [inline docs/index.html]: would be nice if title would be optional like text

File: docs/index.html [hunk 3]
```diff
@@ -25,7 +21,7 @@
 
     */
 
-    var content = { 
+    const content = { 
         defaultImage: "http://i.giphy.com/l0HlRnAWXxn0MhKLK.gif",
```
Comment (mrVanDalo) [inline docs/index.html]: not needed anymore

**Review by mrVanDalo — CHANGES_REQUESTED** [review]
Would be great if you could do these small changes.

File: docs/index.html (hunk 3 above)
Comment (jpkempf) [reply docs/index.html]: how do you mean?

File: docs/index.html (hunk 1 above)
Comment (jpkempf) [reply docs/index.html]: just an innocent test string. sure, i can change it.

File: docs/index.html (hunk 2 above)
Comment (jpkempf) [reply docs/index.html]: can do

File: docs/index.html (hunk 3 above)
Comment (mrVanDalo) [reply docs/index.html]: you dont use it anymore. there have to be a link now.

Comment (jpkempf) [reply docs/index.html]: sorry, still don't understand :[

Comment (mrVanDalo) [reply docs/index.html]: `            if (elem.image) { 
￼                 var image = elem.image;
￼             } else {
￼                 var image = content.defaultImage;
￼             }`

There was a default image before, but you don't use it anymore. That's ok, the `defaultImage` can to be removed now.

Comment (jpkempf) [reply docs/index.html]: ah. yeah, i wasn't sure if you still needed/wanted that for something. okay, gonna remove it.

**Pull request merged**